README.md

```diff
@@ -63,13 +63,17 @@ $ npm install universal-module-tree
 
 ## API
 
-### getTree(dir) => `Promise`
-### getTree.fromPackageLock({ packageLock, packageJSON }) => `Object`
-### getTree.fromYarnLock({ yarnLock, packageJSON }) => `Object`
-### getTree.fromNodeModules(path) => `Promise`
+### getTree(dir, { dev = true }) => `Promise`
```

Comment:
booleans that default to true should really be renamed and default to false aka `skipDev = false`